google/cloud/bigquery/schema.py

```diff
@@ -252,6 +311,18 @@ def max_length(self):
         """Optional[int]: Maximum length for the STRING or BYTES field."""
         return self._properties.get("maxLength")
 
+    @property
+    def range_element_type(self):
+        """Optional[FieldElementType]: The subtype of the RANGE, if the
+        type of this field is RANGE.
+
+        Must be set when ``type`` is `"RANGE"`. Must be one of `"DATE"`,
+        `"DATETIME"` or `"TIMESTAMP"`.
+        """
+        if self._properties.get("rangeElementType"):
+            ret = self._properties.get("rangeElementType")
+            return FieldElementType(element_type=ret["type"])
```

Comment:
should this be something like `FieldElementType.from_api_repr(ret)`?  If the type gets more fields over time you'll possibly lose fidelity since you're only grabbing a single field.

Reply:
Good point! I'll change this

Reply:
done